UserBubble component

violoncelloCH opened the pull request:
introduces a new UserBubble component to be used e.g. for @-mentions in social or talk
it features an optional popover to e.g. display the full username-handle for remote users in social

it should also be possible to link the UserBubble to the user's profile (social), but I'd like to have this optional, but didn't manage to do this, because the `<a>` element would need to wrap the whole bubble and therefore I can't just use `v-if` on it. Any idea @skjnldsv or @ma12-co ?

ma12-co commented:
@violoncelloCH  In talk we can also mention a conversation so we need also the capability to display an icon instead of the avatar component in the bubble, could you add this?

violoncelloCH commented:
thanks for the suggestion @ma12-co !
I've implemented this as a slot (name=avatar), which is by default filled with the avatar... and I also added the possibility to conditionally have a link on the bubble and also the popover is now completely disabled if there is no content in it

skjnldsv commented on src/components/UserBubble/UserBubble.vue:
```diff
@@ -0,0 +1,146 @@
+	display: flex;
+	height: 20px;
+	border-radius: 10px;
+	background-color: var(--color-background-dark);
```
```suggestion
	background-color: var(--color-background-dark);
	h6 {
		line-height: 20px;
	}
```

violoncelloCH replied:
implemented!

skjnldsv commented on src/components/UserBubble/UserBubble.vue:
```diff
@@ -0,0 +1,146 @@
+	<Popover trigger="hover focus" :open="open" :disabled="popoverEmpty"
+		class="user-bubble-popover" @update:open="onOpenChange">
+		<div slot="trigger" v-bind="linkOrNot" class="user-bubble">
+			<slot name="avatar">
```
Let's use a prop and detect if it's a url instead of an avatar
So it can be a css class or just a plain url

violoncelloCH replied:
implemented!
